ReactiveXComponent/Parser/XCApiConfigParser.cs

```diff
@@ -114,43 +120,40 @@ private void AddTopicToRepository(Dictionary<TopicIdentifier, string> repository
 
             if (!repository.ContainsKey(topicIdentifier))
             {
-                repository.Add(topicIdentifier, node?.Descendants(_xc + XCApiTags.Topic).FirstOrDefault().Value);
+                repository.Add(topicIdentifier, node?.Descendants(_xc + XCApiTags.Topic).FirstOrDefault()?.Value);
             }
         }
 
         private Dictionary<long, string> CreateSnapshotTopicByComponentRepository(IEnumerable<XElement> snapshotNodes)
```

Comment:
Same problem with namings here : "Repo" and "Repository"

Reply:
done